h2o-py/h2o/frame.py

```diff
@@ -126,7 +126,7 @@ def from_python(python_obj, destination_frame=None, header=0, separator=",", col
         assert_is_type(separator, I(str, lambda s: len(s) == 1))
         assert_is_type(column_names, None, [str])
         assert_is_type(column_types, None, [coltype])
-        assert_is_type(na_strings, None, [str])
+        assert_is_type(na_strings, None, [str], [[str]], { str: [str, None] })
```

Comment:
Hi William, thanks for your contribution!
The type `{str: [str, None]}` denotes a dictionary where keys are strings, and values are arrays of either strings or None values. The "None value" however doesn't make sense in this context: `na_strings` can be "... or a dictionary of column names to strings which are to be interpreted as missing values". In view of this, could you please change the definition of the last type to be `{str: [str]}`?